### ISSUE TITLE
Error on HPC

### ISSUE THREAD

**Issue by varsh1090** [zhoulab/p53-chip-seq-go#1]
Is this the right method to run this script?

[v.sundaresan@login2 p53-chip-seq-go]$ ./run.sh
Traceback (most recent call last):
  File "Assignment2.py", line 8, in <module>
    from bs4 import BeautifulSoup
ImportError: No module named bs4
[v.sundaresan@login2 p53-chip-seq-go]$ pwd
/ufrc/zhou/share/projects/bioinformatics/p53/P53_ChIPSeq_GO/p53-chip-seq-go

**Comment by victorlin** [comment]
Yes, I updated the environment so it works now. Also made a few changes to adapt to dependency changes over time.
